nautobot/core/templatetags/helpers.py

```diff
@@ -616,6 +616,48 @@ def render_uptime(seconds):
     )
 
 
+@library.filter()
+@register.filter()
+def hyperlinked_field(value, hyperlink):
+    """Render a value as a hyperlink."""
+    if not value:
+        return placeholder(value)
+    return format_html('<a href="{}">{}</a>', value, hyperlink)
```

Comment:
Might make sense for `hyperlink` to be optional and just default to using `value` as both the text and the hyperlink?